pphtml.py

```diff
@@ -1007,19 +1045,25 @@ def find_defined_CSS(self):
         t = [] # place to build a CSS block
         i = 0
 
-        while i < len(self.wb) and not bool(re.search(r'style.*?type.*?text.*?css', self.wb[i])):
-            i += 1 # advance to <style type="text/css"> line
+        # advance to <style type="text/css"> or <style> line for HTML5
+        while i < len(self.wb) \
```

Comment:
Line continuation characters aren't pythonic and it's recommended to use `()`s and retain the 4-space indents. 
```python
        while (i < len(self.wb) 
            and not bool(re.search(r'style.*?type.*?text.*?css', self.wb[i]))
            and not bool(re.search(r'<style>', self.wb[i]))):
            i += 1
```
Also you shouldn't need `bool()` to test the truthiness of `re.search` but there's no harm in it.

Ditto the change below on line 1063.

Reply:
Thanks, Casey. I'll make those changes. I'm guessing I can do that to the file where it sits, or once there is a PR maybe I have to do a new PR? Still learning.

Reply:
No I guess I can't edit it where it sits because that's in DIstributedProofreaders. So do I I change it over in asylumcs/pphtml and do another PR?

Reply:
Ok Casey I think the corrected code is in place (thanks to srjfoo walking me through it).
Also there are two open issues for pphtml that have long been addressed. Can they be closed?